### PR TITLE
PHPUnit tests

### PR DESCRIPTION
**Pull request by cadic**
### Description of the Change

Adds PHP Unit tests

### Changelog Entry

Added - Unit tests

### CODE REVIEW

**Comment by jeffpaul** [comment]
@cadic what else remains here to open this for review/merging?  Not looking to get full coverage, just getting something here would be good.

**Comment by jeffpaul** [comment]
@cadic pinging again here to see if there's more to get this up for review/merge?

File: phpunit.xml.dist [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<phpunit
+		bootstrap="tests/unit/bootstrap.php"
+		backupGlobals="false"
+		colors="true"
+		convertErrorsToExceptions="true"
+		convertNoticesToExceptions="true"
+		convertWarningsToExceptions="true"
+		verbose="true"
```
Comment (faisal-alvi) [inline phpunit.xml.dist]: seems like an extra tab?

**Review by faisal-alvi — DISMISSED** [review]

File: phpunit.xml.dist (hunk 1 above)
Comment (cadic) [reply phpunit.xml.dist]: @faisal-alvi thank you, fixed that in 556b11e003f2df130ce62ed73809df2a3a6cb459

**Review by faisal-alvi — APPROVED** [review]
🚀

**Pull request merged**